app/styles/app.scss

```diff
@@ -556,12 +558,13 @@ hr {
             opacity: 1;
         }
     }
-    .row {
-        margin: 0;
-    }
-    label, input, textarea {
-        display: block;
-    }
+    // THESE STYLES ARE INTERFERING WITH MY STYLES. 
```

Comment:
I'm just going to go out on a limb and assume this will be resolved before merging. For example, by using more distinctive style names, namespaced to your specific component.